ansible/roles/pg/tasks/configure/create_standby.yml

```diff
@@ -27,8 +27,9 @@
   become: true
 
 # https://www.repmgr.org/docs/current/quickstart-standby-clone.html
+# TODO: maybe force '-F' should not be used lightly
```

Comment:
Can you add some more context here?

Reply:
Actually, I think it's fine using force both here and for standby register, since this `yml` file is only called when we first set up the VMs or when we need to rebuild a replica after the migrations. So it makes sense simply removing the TODO

Reply:
Removed the TODO and FIXME